### DISCUSSION TITLE
Is there a forum for Ultragrid users somewhere?

### DISCUSSION THREAD

**Discussion by ygreq** [CESNET/UltraGrid#141] [General]
Hi there! 

Is there a forum for Ultragrid users to exchange ideas and troubleshoot problems? Or should we consider this to be a forum? :)

I will start asking questions here. If they will be considered spam or anything else please delete. But please let me know where one should find a forum for Ultragrid. 

Thank you so much for this wonderful piece of software. 
ygreq

**Comment by alatteri** [comment]
I think this is good as any.
